jetson/nucleo_bridge/Hardware.h

```diff
@@ -0,0 +1,109 @@
+#ifndef HARDWARE_H
+#define HARDWARE_H
+
+#include <algorithm>
+
+enum HardwareType
```

Comment:
Please add a comment for the enum like you have for classes.

Reply:
done